### DISCUSSION TITLE
系统已经设置过默认英文了，可是列表页的搜索功能却还是默认中文

### DISCUSSION THREAD

**Discussion by zz99zz9** [ant-design/ant-design#41531] [General]
问题如题，不知道如何解决？是不是搜索功能需要有单独配置默认英文的地方啊

**Comment by zombieJ** [comment]
搞个 Codesandbox 看看？

**Reply by zz99zz9** [reply]
<img width="1507" alt="image" src="https://user-images.githubusercontent.com/22146191/228481579-a151e42b-32a1-44bf-ab4f-ed3a0f4e56d5.png">
